src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
```

Comment:
If we call `ConnectivityIndicator.getInstance.show()` on `onActivityResumed` if we are on editing mode we won't have to initialize it unnecessarily here.